packages/manager/apps/pci-project/src/components/payment/integrations/PaypalPaymentMethodIntegration.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useImperativeHandle, useMemo } from 'react';
+import { TCart } from '@/data/types/payment/cart.type';
+import {
+  TPaymentMethod,
+  TPaymentMethodIntegrationRef,
+  TRegisterPaymentMethod,
+} from '@/data/types/payment/payment-method.type';
+
+import { PayPalButton } from './paypal/components/PayPalButton';
+import { usePayPalPayment } from './paypal/hooks/usePayPalPayment';
+import { usePayPalSDK } from './paypal/hooks/usePayPalSDK';
+
+interface PaypalPaymentMethodIntegrationProps {
+  handleCustomSubmitButton?: (btn: string | JSX.Element) => void;
+  paymentHandler: React.Ref<TPaymentMethodIntegrationRef>;
+  isSetAsDefault?: boolean;
+  onSuccess?: () => void;
```

Comment:
`onSuccess` does not exist in PaymentMethodIntegration. Do you mean to use `onPaymentSubmit` to go through the payment process?
`onError` does not exist either but in my (future) PR I added a `onPaymentError` callaback to call in case of error so no need to update it now, it will be in my PR ;)